pkgs/games/openmw/default.nix

```diff
@@ -11,14 +11,14 @@ let
     };
   });
 in stdenv.mkDerivation rec {
-  version = "0.43.0";
+  version = "0.44.0";
   name = "openmw-${version}";
 
   src = fetchFromGitHub {
     owner = "OpenMW";
     repo = "openmw";
-    rev = name;
-    sha256 = "1nybxwp77qswjayf0g9xayp4x1xxq799681rhjlggch127r07ifi";
+    rev = "ef85b1393aeaec905f6310abca29b4e0f42db901";
```

Comment:
This should use the openmw-0.44.0 tag which corresponds to that hash for clarity.

Reply:
whoops, I forgot about this. I've got to review packaging + github docs I think. I'm going to refrain from going near master until I get a bit more familiar, sorry about that & thanks for the the review tip.